e2e/testsuite/testsuite.go

```diff
@@ -611,6 +628,43 @@ func (s *E2ETestSuite) QueryGranterGrants(ctx context.Context, chain *cosmos.Cos
 	return grants.Grants, nil
 }
 
+// IsRelayer returns true if the provided wallet is used by a relayer.
+func (s *E2ETestSuite) IsRelayer(user ibc.Wallet) bool {
+	if s.relayers == nil {
+		return false
+	}
+	return s.relayers[user]
+}
+
+// retryNtimes retries the provided function up to the provided number of attempts.
+func (s *E2ETestSuite) retryNtimes(f func() (sdk.TxResponse, error), attempts int) (sdk.TxResponse, error) {
+	// Ignore account sequence mismatch errors.
+	allowedMessages := []string{"account sequence mismatch"}
```

Comment:
Since this is a configuration parameter, maybe we can move it to the top [together with the `const`](https://github.com/cosmos/ibc-go/blob/main/e2e/testsuite/testsuite.go#L48-L55)? We can make it an unexpected `var`.